### ISSUE TITLE
Additional instructions for SPServiceInstance.Unprovision Method

### ISSUE THREAD

**Issue by lindalu-MSFT** [SharePoint/sp-dev-docs#2675]
Note: You should run this command only from the server on which you would like to Unprovision the service instance.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4a61a2aa-988e-c10b-53fa-cf5e9ec42630
* Version Independent ID: 1f1c90f3-31d1-80d5-e5f6-db42a7970551
* Content: [SPServiceInstance.Unprovision Method (Microsoft.SharePoint.Administration)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.sharepoint.administration.spserviceinstance.unprovision?redirectedfrom=MSDN&view=sharepoint-server#Microsoft_SharePoint_Administration_SPServiceInstance_Unprovision)
* Content Source: [sharepoint-server/xml/Microsoft.SharePoint.Administration/SPServiceInstance.xml](https://github.com/SharePoint/sp-dev-docs-2013-server-ref-dotnet/blob/live/sharepoint-server/xml/Microsoft.SharePoint.Administration/SPServiceInstance.xml)
* Product: **sharepoint**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **o365devx**

**Comment by thechriskent** [comment]
Good note, but closing as an issue. Thanks!

**Comment by lindalu-MSFT** [comment]
We'd like to get this change (and the other three) into the reference so it doesn't get overwritten each time it's autogen'd.  We can't do it the "normal" github way by making and edit then opening a PR, because it will just get overwritten which is why I added it as a Comment so customers can see it. What's the best way to make these four changes part of the permanent reference?  Thanks! Linda Lu
p.s. I have a bug for this that I just assigned to @VesaJuvonen. Let me know if there's someone else I should assign it to!
